lib/tugboat/config.rb

```diff
@@ -16,7 +16,7 @@ class Configuration
     DEFAULT_IMAGE = 'ubuntu-14-04-x64'
     DEFAULT_SIZE = '512mb'
     DEFAULT_SSH_KEY = ''
-    DEFAULT_IP6 = 'false'
+    DEFAULT_IP6 = 'true'
```

Comment:
Can we leave this as false by default? Although IP6 isn't that destructive, the default state for both private networking and ip6 in DO is false, so I think we should stick with that :+1: